packages/office-ui-fabric-react/src/components/pickers/BasePicker.Props.ts

```diff
@@ -88,4 +105,24 @@ export interface IBasePickerSuggestionsProps {
    * The text to display while the results are loading.
    */
   loadingText?: string;
+  /**
+   * The text to display while searching for more results in a limited sugesstions list.
+   */
+  searchingText?: string;
+  /**
+   * Optional text shown at the end of the suggestions list when the list when there are more items than then number specified by resultsMaximumNumber.
+   */
+  resultsFooterFull?: string;
+  /**
+   * Optional text shown at the end of the suggestions list when all relevant items are currently visible.
+   */
+  resultsFooter?: string;
```

Comment:
I tentatively think that ` resultsFooter ` and ` resultsFooterFull ` should be some sort of IRenderFunction so that the user can customize their footer more.